packages/crab-netting/src/CrabNetting.sol

```diff
@@ -7,49 +9,126 @@ contract CrabNetting {
     address usdc;
     address crab;
 
-    mapping(address => uint256) usd_balance;
-    mapping(address => uint256) crab_balance;
+    mapping(address => uint256) public usd_balance;
+    mapping(address => uint256) public crab_balance;
     struct Receipt {
-        address depositor;
+        address sender;
         uint256 amount;
     }
-    Receipt[] deposits;
-    Receipt[] crab_deposits;
+    Receipt[] public deposits;
```

Comment:
`deposits.length` is not possible from UI. So we need the last index to loop and get receipts to show it to admin.

Reply:
depositsQueued and withdraws Queued does this for you